ee/clickhouse/queries/cohort_query.py

```diff
@@ -438,10 +459,10 @@ def get_performed_event_condition(self, prop: Property, prepend: str, idx: int)
 
         self._check_earliest_date((date_value, date_interval))
 
-        field = f"{'NOT' if prop.negation else ''} countIf(timestamp > now() - INTERVAL %({date_param})s {date_interval} AND timestamp < now() AND {entity_query}) > 0 AS {column_name}"
+        field = f"countIf(timestamp > now() - INTERVAL %({date_param})s {date_interval} AND timestamp < now() AND {entity_query}) > 0 AS {column_name}"
         self._fields.append(field)
 
-        return column_name, {f"{date_param}": date_value, **entity_params}
+        return f"{'NOT' if prop.negation else ''} {column_name}", {f"{date_param}": date_value, **entity_params}
```

Comment:
Super strange. `test_unwrap_multiple_levels` was failing if the negation was inlined with the countif. 

@rcmarron any idea why there might be a difference if the 'NOT' is in the SELECT vs the WHERE?